lib/open_feature/sdk/evaluation_context.rb

```diff
@@ -0,0 +1,21 @@
+module OpenFeature
+  module SDK
+    class EvaluationContext
+      TARGETING_KEY = "targeting_key"
+
+      attr_reader :fields
+
+      def initialize(targeting_key: nil, **fields)
+        @fields = {TARGETING_KEY => targeting_key}.merge(fields)
```

Comment:
Since `targeting_key` is the only called out field in the spec, I think we should explicitly add it each time, even if it's  `nil`.